### PR TITLE
Fix submodel instances due to lack of support in OSPRay; cleanup create- and addModel usage

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

**Comment by favreau** [comment]
For the record: https://github.com/ospray/ospray/issues/256

File: plugins/engines/ospray/OSPRayScene.cpp [hunk 1]
```diff
@@ -110,6 +110,11 @@ void OSPRayScene::commit()
                 BRAYNS_INFO << "Committing " << modelDescriptor->getName()
                             << std::endl;
                 _addInstance(_rootModel, impl.getModel(), transformation);
+
+#ifndef OSPRAY_SUPPORTS_SUBMODEL_INSTANCES
```
Comment (favreau) [inline plugins/engines/ospray/OSPRayScene.cpp]: From my understanding, OSPRAY_SUPPORTS_SUBMODEL_INSTANCES will apparently never be true :( Do you still want to keep it?

Comment (tribal-tec) [reply plugins/engines/ospray/OSPRayScene.cpp]: After I read the ticket of course not :) I'm about to push that.

**Comment by favreau** [comment]
Super cool PR overall, looking forward to using the new feature :)

**Pull request merged**